index.js

```diff
@@ -0,0 +1,106 @@
+const express = require('express');
+const helmet = require('helmet');
+const knex = require('knex')
+const server = express();
+const dbConfig = require('./db/knexfile')
+const bcrypt = require('bcryptjs');
+
+const db = knex(dbConfig.development);
+
+server.use(express.json());
+server.use(helmet());
+
+// endpoints here
+
+server.get('/', (req, res) => {
+    res.send('Api Online')
+})
+
+server.get('/api/user', (req, res) => {
+    db('auth')
+    .select('id', 'username', 'password')
+        .then(user => {
+            res.status(200).json(user)
+        })
+        .catch(err => res.status(500).json(err))
```

Comment:
You know the drill, but stick a console.log in here. More knowledge == More power.